### PR TITLE
Update external link icons

### PR DESCRIPTION
**Pull request by bricelam**

### CODE REVIEW

File: entity-framework/index.md [hunk 1]
```diff
@@ -272,7 +272,7 @@ uid: index
                                                     </div>
                                                 </div>
                                                 <div class="cardText">
-                                                    <h3>⤤ API Reference</h3>
+                                                    <h3>API Reference <span class="docon docon-navigate-external" aria-hidden="true" /></h3>
                                                     <p>Browse the Entity Framework 6 API, organized by namespace.</p>
```
Comment (bricelam) [inline entity-framework/index.md]: 😁

image

File: entity-framework/toc.yml [hunk 2]
```diff
@@ -569,5 +569,5 @@
             href: ef6/resources/licenses/ef6/kor.md
           - name: Russian
             href: ef6/resources/licenses/ef6/rus.md
-    - name: "⤤ EF6 API Reference"
+    - name: "EF6 API Reference <span class=\"docon docon-navigate-external\" aria-hidden=\"true\" />"
       href: https://docs.microsoft.com/dotnet/api/?view=entity-framework-6.2.0
```
Comment (bricelam) [inline entity-framework/toc.yml]: 😭

image

File: entity-framework/index.md [hunk 3]
```diff
@@ -211,7 +211,7 @@ uid: index
                                             </div>
                                             <div class="cardText">
                                                 <h3>
-                                                    <a href="https://docs.microsoft.com/dotnet/api/?view=efcore-2.1">⤤ API Reference</a>
+                                                    <a href="https://docs.microsoft.com/dotnet/api/?view=efcore-2.1">API Reference <span class="docon docon-navigate-external" aria-hidden="true" /></a>
```
Comment (mairaw) [inline entity-framework/index.md]:
```suggestion
                                                    <a href="https://docs.microsoft.com/dotnet/api/?view=efcore-3.0">API Reference <span class="docon docon-navigate-external" aria-hidden="true" /></a>
```

File: entity-framework/toc.yml [hunk 4]
```diff
@@ -261,7 +261,7 @@
         href: core/miscellaneous/cli/dbcontext-creation.md
       - name: Design-time Services
         href: core/miscellaneous/cli/services.md
-    - name: "⤤ EF Core API Reference"
+    - name: "EF Core API Reference <span class=\"docon docon-navigate-external\" aria-hidden=\"true\" />"
       href: https://docs.microsoft.com/dotnet/api/?view=efcore-2.1
```
Comment (mairaw) [inline entity-framework/toc.yml]:
```suggestion
      href: https://docs.microsoft.com/dotnet/api/?view=efcore-3.0
```

File: entity-framework/toc.yml (hunk 2 above)
Comment (mairaw) [reply entity-framework/toc.yml]: Not sure how to solve this. Perhaps just remove the icon in this case?

File: entity-framework/index.md (hunk 3 above)
Comment (smitpatel) [reply entity-framework/index.md]: or perhaps URL with latest.

**Comment by bricelam** [comment]
Updated to the ">>" convention used in other docs.

**Pull request merged**